cli/client/print.go

```diff
@@ -18,13 +18,21 @@ var PrintMessage = printMessage
 // PrintMessage() before exiting to allow assertions against captured output.
 var PrintMessageAndExit = printMessageAndExit
 
+// Exit is a placeholder function that wraps a call to os.Exit to allow
+// assertions against exit with an exit code.
+var Exit = exit
```

Comment:
@elezar Why the indirection on this?

Reply:
The unit tests now check for a non-zero status code, but there is some discussion in  #1664 on whether this is needed.